src/main/java/com/sollace/coppergolem/entity/ai/PositionFinder.java

```diff
@@ -54,7 +53,7 @@ public Optional<BlockPos> pickAny(Predicate<BlockPos> filter) {
     }
 
     private Stream<BlockPos> findPositions() {
-        knownPositions.removeIf(p -> !entity.getEntityWorld().isChunkLoaded(p) || !isValid(toState(p)));
+        knownPositions.removeIf(p -> !entity.getEntityWorld().isChunkLoaded(p.getX(), p.getZ()) || !isValid(toState(p)));
```

Comment:
What is the purpose of this change?

Reply:
The deprecation annotations come from Mojang, and the way they use it is... A little weird. It doesn't mean they're going to remove, but just that it's not meant to be called directly.

It's like the methods in Block and Item, where deprecated just means overriding/implementing is fine, but calling usually goes through BlockState/ItemStack. It's not a big deal, so I guess this change can stay. Just remember that for other changes moving forward.